model.py

```diff
@@ -0,0 +1,281 @@
+from typing import Dict
+
+from prometheus_client import Gauge
+
+
+class DataCategoryConstants:
+    Index = "Index"
+    Operator = "Operator"
+    Query = "Query"
+    QueryType = "QueryType"
+    Session = "Session"
+    Snapshot = "Snapshot"
+    Stream = "Stream"
+    Transaction = "Transaction"
+    Trigger = "Trigger"
+    General = "General"
+
+
+INDEX = {
+    "ActiveLabelIndices": Gauge(
+        "ActiveLabelIndices", "Number of active label indices in the system."
```

Comment:
Did you use style guide here or formatter, maybe it makes sense to run it through file?